packages/react/src/components/Toast/index.tsx

```diff
@@ -105,6 +109,7 @@ export default class Toast extends React.Component<ToastProps, ToastState> {
       toastRef,
       focus,
       show,
+      dismissible,
```

Comment:
It took me a minute to see that the default is `true`. I haven't seen/used `defaultProps` (or `class` components) in a really, really long time 😅

Reply:
Yep, haha. It's fun updating things in three different places.